config/rules.php

```diff
@@ -71,4 +71,14 @@
             'different:current_password',
         ],
     ],
+
+     /*
+     * User Threads Validation Rules.
+     */
+    'thread' => [
+        'title' => ['required'],
+        'body' => ['required'],
+        'user_id' => ['required'],
+        'channel_id' => ['required'],
+    ],
 ];
```

Comment:
Can you add more validation rules here? like `['title' => ['required', 'string', 'max:255']]`

Reply:
You can add these to the `id` feilds: `['required', 'numeric', 'exists:users,id']`. Same goes to the channel.